src/policy/immix/mod.rs

```diff
@@ -16,4 +16,4 @@ pub const BLOCK_ONLY: bool = false;
 
 /// Mark lines when scanning objects.
 /// Otherwise, do it at mark time.
-pub const MARK_LINE_AT_SCAN_TIME: bool = true;
+pub const MARK_LINE_AT_SCAN_TIME: bool = false;
```

Comment:
Concurrent marking does not have scan object packet any more, for simplicity, I just change this value

Reply:
This is reverted back to `true`. I added `post_scan_object` in `ConcurrentTraceObject` when a object is scanned. `post_scan_object` should be called when an object gets scanned regardless of how Immix space scans objects.